view/frontend/web/template/payment/amwal-payment/form.html

```diff
@@ -42,7 +42,7 @@
                data-bind="attr: {'id': getCode()}, value: getCode(), checked: isChecked, click: selectPaymentMethod, visible: isRadioButtonVisible()"/>
         <label data-bind="attr: {'for': getCode()}" class="label">
             <div class="payment-brand">
-                <span data-bind="text: getTitle()"></span>
+                <span id="payment-label"></span>
```

Comment:
If we're using a fixed string you could do
```suggestion
                <span data-bind:"i18n: 'Quick checkout (Amwal)'"></span>
```

This way you wont need to introduce a new method to update the text. Alternatively you could also choose to override the `getTitle()` method in the amwal-payment.js file, or update the title in `\Amwal\Payments\Model\Config\Checkout\ConfigProvider::getConfig()`.